src/Illuminate/Cache/Repository.php

```diff
@@ -188,7 +188,9 @@ public function pull($key, $default = null)
     public function put($key, $value, $minutes = null)
     {
         if (is_array($key)) {
-            return $this->putMany($key, $value);
+            $this->putMany($key, $value);
+
+            return;
```

Comment:
This shouldn't be void as it implements PSR-16 https://github.com/php-fig/simple-cache/blob/master/src/CacheInterface.php#L77

Reply:
we're talking about `put` method which is not on that interface, or am I missing something?

Reply:
@deleugpn this might be how we can make it adhere to the interface in question without more in-depth analysis/changes:

![image](https://user-images.githubusercontent.com/6928818/42619387-7919d48e-85ea-11e8-9929-9361be4240a7.png)


does it make sense?

Reply:
Nevermind. I just noticed that the method is already void, it's just not void if we pass an array that ends up being passed to `putMany`. The PSR-16 implementation is kind of flawed already anyway, so no point in dealing with that in this PR.